### PR TITLE
Fix background width

### LINKED ISSUE #309: New SEM Pages - Content Rendering Issues
Main Content section not rendering properly. Need to add inline form for two column layout. Ensure icons and text lay cleanly with background image below (currently overlapping) 

image

Should look like:
image

### LINKED ISSUE #311: Testimonial Styling Issues
Testimonial Quote styling is off – should have orange bar and text should be larger 

Example: https://main--shredit--stericycle.aem.live/en-us/marketing/bulk-paper-shredding 

image

image

### LINKED ISSUE #313: New SEM Page - Shred-it By The Numbers Layout
Shred-it By The Numbers – Form should be side by side with content in 2 column layout

image

### LINKED ISSUE #315: New SEM Pages - Margin Issues
Margins on new SEM page do not match live site (page is too narrow)

Examples: https://main--shredit--stericycle.aem.live/en-us/marketing/bulk-paper-shredding and https://main--shredit--stericycle.aem.live/en-us/marketing/hard-drive-destruction-service (this applies to all pages using this layout)

image
image

### LINKED ISSUE #316: Marketing Footer Rendering Issues
**New Marketing Page**

Example: https://main--shredit--stericycle.aem.live/en-us/marketing/hard-drive-destruction-service 

Lots of extra spacing, random extra phone and get a quote at top, horizontal divider uneeded. 

image

Should be:
image

Other Marketing Pages:

Example: https://main--shredit--stericycle.aem.live/en-us/marketing/campaign-page-optimized

Showing extra vertical divider and unnecessary vertical space  

image

### LINKED ISSUE #319: New SEM Page Inline Issues
Example: https://main--shredit--stericycle.aem.live/en-us/marketing/shred-it-document-destruction-inline 
o	:shred-it-a-20stericycle-20solution-logo-white: showing on the page
o	Form should be inline to the right of the content not below
image
o	25 Off Badge and text missing 
image

### LINKED ISSUE #320: eComm Marketing Page Issues
Example: https://main--shredit--stericycle.aem.live/en-us/marketing/ecomm
o	Icon for Select Is Missing
o	Icons are too large
o	CTAs under each section are missing 

Should look like:
image

### LINKED ISSUE #336: Industry Pages - Gray Background
Gray background needs to be applied to Recommended for sections
o	Example: https://main--shredit--stericycle.aem.live/en-us/who-we-serve/accountants 

This applies to all Who We Serve pages

image

### LINKED ISSUE #414: Cards - Rendering Issues
Just reviewed Card components. There are two with Rendering Issues.

1. https://main--shredit--stericycle.aem.page/tools/sidekick/library.html?plugin=blocks&path=/tools/sidekick/blocks/cards&index=4

Layout/Styling appears like a jumbled mess

image

2. https://main--shredit--stericycle.aem.page/tools/sidekick/library.html?plugin=blocks&path=/tools/sidekick/blocks/cards&index=6

- Bullets shouldn't be indented
- Add line spacing between bullets and subcopy

image

### PR DESCRIPTION
**Pull request by MarquiseRosier**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #315 
Fix #309 
Fix #311 
Fix #313 
Fix #315 
Fix #316 
Fix #319 
Fix #320 
Fix #336 
Fix #414

Test URLs:
- Before: 
  - https://main--shredit--stericycle.aem.page/en-us/marketing/bulk-paper-shredding
  - https://main--shredit--stericycle.aem.page/en-us/marketing/hard-drive-destruction-service
- After: 
  - https://fix-background-width--shredit--stericycle.aem.live/en-us/marketing/hard-drive-destruction-service
  - https://fix-background-width--shredit--stericycle.aem.live/en-us/marketing/bulk-paper-shredding

### CODE REVIEW

**Comment by bstopp** [comment]
Found some issues with this. Providing some fixes.

**Comment by MarquiseRosier** [comment]
@bstopp thanks a ton! :)

**Review by jindaliiita — APPROVED** [review]
Need to discuss the CLS part

File: scripts/scripts.js [hunk 1]
```diff
@@ -517,9 +517,9 @@ async function loadEager(doc) {
   const main = doc.querySelector('main');
   if (main) {
     decorateMain(main);
+    await decorateTemplates(main);
     document.body.classList.add('appear');
     await loadSection(main.querySelector('.section'), waitForLCP);
-    await decorateTemplates(main);
```
[the remark this answers is not in the document]
Comment (bstopp) [reply scripts/scripts.js:522]: The templates that contain a form need to be reevaluated or redesigned - the form and how it is added to the page contributes to either massive CLS, or terrible LCP

[the remark this answers is not in the document]
Comment (jindaliiita) [reply scripts/scripts.js:522]: Totally agree with this

[the remark this answers is not in the document]
Comment (jindaliiita) [reply scripts/scripts.js:536]: @bstopp Just wanting to understand if you remember why the position of decorateTemplates has changed. Currently, it's leading to a significant CLS issue, and ideally, it should be after the await loadSection(main.querySelector('.section'), waitForLCP);. We should ideally load the LCP block section first.
Sample Page - https://main--shredit--stericycle.aem.page/en-us/express-priority-shredding-services

**Pull request merged**